lab-gurpreet/bin/async_read.js

```diff
@@ -0,0 +1,14 @@
+var async = require('../lib/async_work');
```

Comment:
You have your CLI in your bin file, which indicates that it is a binary file (executable), but you haven't actually made it executable yet.